lesson_2/Lesson_2.1.ipynb

```diff
@@ -0,0 +1,375 @@
+{
```

Comment:
<p>It's a section now, not a segment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>